utils/frame/remote-externalities/src/lib.rs

```diff
@@ -428,81 +564,61 @@ where
 		let (tx, mut rx) = mpsc::unbounded::<Message>();
 
 		for thread_keys in keys_chunked {
-			let thread_client = client.clone();
 			let thread_sender = tx.clone();
+			let thread_client = client.clone();
```

Comment:
Maybe this could an issue that the underlying WebSocket connection task is shared by all messages but yeah I have run this myself.